app/assets/javascripts/discourse/app/routes/discourse.js

```diff
@@ -66,11 +67,8 @@ const DiscourseRoute = Route.extend({
     return user.id === this.currentUser.id;
   },
 
-  isPoppedState(transition) {
-    return (
-      !transition._discourse_intercepted &&
-      (!!transition.intent.url || !!transition.queryParamsOnly)
-    );
+  isPoppedState() {
```

Comment:
Can we convert it to a getter or is any external code relying on this?

Reply:
Only one private plugin using it. TBH it's pretty weird to have this on the route base class, so let's delete it and make people inject the new history-store service instead 🔥